### DISCUSSION TITLE
is there a way to inline css modules in the head

### DISCUSSION THREAD

**Discussion by yanv1991** [vercel/next.js#13820] [Help]
I'm working on an project that uses legacy css modules so migrate all those styles to a styled-component solution is not an option, so I wonder if there is a way to make this as style-loader does, btw I tried with the loader, but didn't work.

Also my css assets are stored in an external source, so this [solution](https://github.com/vercel/next-plugins/issues/238#issuecomment-432211871) didn't work to me.

**Comment by yangzhennupt** [comment]
I have same problem， import componets from node_modules， I want to inject css in <head> with style tag (like style-loader), not extract css file then fetch in client, any solutions?

**Comment by jeff-zhenz** [comment]
+1, wait for an elegant solution
